### ISSUE TITLE
Is there a way to pass a GitHub API Token?

### ISSUE THREAD

**Issue by ob** [google/copybara#101]
I am getting rate limited by GitHub when moving a bunch of commits:

```
% copybara copy.bara.sky export_to_github_pr
Nov 06, 2019 4:16:36 PM com.google.copybara.Main configureLog
INFO: Setting up LogManager
Copybara source mover (Version: Unknown version)
...
ERROR: Migration of origin revision '<commit_sha1>' failed with error: GitHub API call failed with code 403 The request was GET <redacted> Response:
{"message":"API rate limit exceeded for <ip_address>. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)","documentation_url":"https://developer.github.com/v3/#rate-limiting"}
```

**Comment by hsudhof** [comment]
As discussed in #54, git-credential files should be respected. This was broken by a regression, fixed in e63206370dfc793ceb6676be1b17103c66750eb3 (and follup-up)

Cheers,
~H

**Comment by ob** [comment]
Thanks @hsudhof, e632063 fixes this problem!
